src/main/scala/org/camunda/feel/api/FeelEngineBuilder.scala

```diff
@@ -74,3 +77,9 @@ case class FeelEngineBuilder private (
   )
 
 }
+
+object FeelEngineBuilder {
+
+  @javaStatic
+  def create(): FeelEngineBuilder = FeelEngineBuilder()
```

Comment:
:+1: Nice.

Reply:
:wrench: Additionally, we could create a convenient method for Java applications. Usually, every Java application will use the JavaValueMapper. 

For example:

```
def forJava(): FeelEngineBuilder = FeelEngineBuilder().withCustomValueMapper(new JavaValueMapper)
```

Reply:
@saig0 tbh I am not sure I really understood your idea about `forJava` method.. Could you please elaborate?

Reply:
Done